test/Password.ts

```diff
@@ -1,458 +1,254 @@
-      await firstPasswordInput.clear();
-      await secondPasswordInput.clear();
+      await NewAccountScreen.passwordInput.clearValue();
+      await NewAccountScreen.passwordConfirmationInput.clearValue();
     });
 
     it('Minimum password length 8 characters', async function () {
-      await firstPasswordInput.sendKeys(PASSWORD.SHORT);
-      expect(await firstPasswordErrorDiv.getText()).matches(
-        /password is too short/i
+      await NewAccountScreen.passwordInput.setValue(PASSWORD.SHORT);
+      expect(NewAccountScreen.passwordError).toHaveText(
+        'Password is too short'
       );
     });
 
     it('Passwords in both fields must mismatch', async function () {
-      // check password mismatches
-      await firstPasswordInput.sendKeys(PASSWORD.DEFAULT);
-      await secondPasswordInput.sendKeys(PASSWORD.SHORT);
-      expect(await secondPasswordErrorDiv.getText()).matches(
-        /passwords do not match/i
+      await NewAccountScreen.passwordInput.setValue(PASSWORD.DEFAULT);
+      await NewAccountScreen.passwordConfirmationInput.setValue(PASSWORD.SHORT);
+      expect(NewAccountScreen.passwordConfirmationError).toHaveText(
+        'Passwords do not match'
       );
     });
 
     it('Passwords in both fields must match', async function () {
-      await firstPasswordInput.sendKeys(PASSWORD.DEFAULT);
-      await secondPasswordInput.sendKeys(PASSWORD.DEFAULT);
-      expect(await secondPasswordErrorDiv.getText()).to.be.empty;
+      await NewAccountScreen.passwordInput.setValue(PASSWORD.DEFAULT);
+      await NewAccountScreen.passwordConfirmationInput.setValue(
+        PASSWORD.DEFAULT
+      );
+      expect(NewAccountScreen.passwordConfirmationError).toHaveText('');
     });
 
     it('The ability to paste the password from the clipboard');
 
     it('Successful password creation', async function () {
-      await firstPasswordInput.sendKeys(PASSWORD.DEFAULT);
-      await secondPasswordInput.sendKeys(PASSWORD.DEFAULT);
-      await this.driver.findElement(By.css('#termsAccepted')).click();
-      await this.driver.findElement(By.css('#conditionsAccepted')).click();
-      await this.driver
-        .wait(
-          until.elementIsEnabled(
-            this.driver.findElement(By.css('button[type=submit]'))
-          ),
-          this.wait
-        )
-        .click();
-      // check we are at create new account page
-      await this.driver.wait(
-        until.elementLocated(
-          By.xpath("//div[contains(@class, 'root@import')]")
-        ),
-        this.wait
+      await NewAccountScreen.passwordInput.setValue(PASSWORD.DEFAULT);
+      await NewAccountScreen.passwordConfirmationInput.setValue(
+        PASSWORD.DEFAULT
       );
-      expect(
-        await this.driver
-          .findElement(By.css('[data-testid="createNewAccountBtn"]'))
-          .getText()
-      ).matches(/create a new account/i);
+      await NewAccountScreen.termsAndConditionsLine.click();
```

Comment:
I use click on whole line b/c i can find it by text.

Reply:
You use https://testing-library.com/docs/queries/bylabeltext and that way what you find is the checkbox itself. But that really doesn't matter, click is actually been done on the checkbox anyway even if you used `findByText`, because it's a `label` element

Reply:
done